src/sql/workbench/api/common/extHostModelView.ts

```diff
@@ -1544,7 +1544,7 @@ class DeclarativeTableWrapper extends ComponentWrapper implements azdata.Declara
 		super(proxy, handle, ModelComponentTypes.DeclarativeTable, id, logService);
 		this.properties = {};
 		this._emitterMap.set(ComponentEventType.onDidChange, new Emitter<any>());
-		this._emitterMap.set(ComponentEventType.onDidClick, new Emitter<any>());
+		this._emitterMap.set(ComponentEventType.onSelectedRowChanged, new Emitter<any>());
```

Comment:
update this to use the proper event.

Reply:
I think we should consider keeping the original event too - clicking the table is still a valid action (although I guess I'm not exactly sure what a good use case for hooking into that would be)